examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -18,15 +18,59 @@
  */
 
 const app = {
+  isInitialized: false,
   // Application Constructor
   initialize: function() {
     document.addEventListener(
       "deviceready",
       this.onDeviceReady.bind(this),
       false
     );
+    // this method gets called after configure is done
+    // or when the customerInfo is updated from a purchase, restore, login / out or renewal
     window.addEventListener("onCustomerInfoUpdated", (info) => {
+      if (this.isInitialized) { return; }
+      this.isInitialized = true;
 
+      this.setupShouldPurchasePromoProductListener();
+
+      Purchases.enableAdServicesAttributionTokenCollection();
+      Purchases.getCustomerInfo(
+        info => {
+          const isPro = typeof info.entitlements.active.pro_cat !== "undefined";
+          console.log("isPro " + JSON.stringify(isPro));
+          console.log(JSON.stringify(info));
+        },
+        error => {
+          debugger;
+          console.log(JSON.stringify(error));
+        }
+      );
+  
+      Purchases.isAnonymous(
+        isAnonymous => {
+          console.log("ISANONYMOUS " + isAnonymous);
+          }
+      );
+  
+      Purchases.getOfferings(
+        offerings => {
+          Purchases.checkTrialOrIntroductoryPriceEligibility([offerings.current.lifetime.product.identifier, "some_offering"],
+            map => {
+              console.log(map)
+            }
+          );
+          console.log(JSON.stringify(offerings));
+        },
+        ( error ) => {
+          console.log(JSON.stringify(error));
+        }
+      );
+
+      Purchases.setPhoneNumber("12345678");
+      Purchases.setDisplayName("Garfield");
+      Purchases.setAttributes({ "favorite_cat": "garfield" });
+      Purchases.setEmail("garfield@revenuecat.com");
```

Comment:
I guess we could move all this to a `initialize/setup` function if we wanted to make it cleaner. NABD though.

Reply:
I actually plan to re-do most of this app since it's not very useful. I'll share more in slack. Merging for now